src/util/nativeStorageHelper.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable fp/no-class */
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+/**
+ *
+ */
+function handleError(func: string, param?: string): Promise<string> {
+  let message
```

Comment:
I'm not sure why the linter is not catching this, but we try to avoid `let` and mutations. This expression can easily be rewritten with a ternary statement.